instrumentation/lettuce/lettuce-5.1/testing/src/main/groovy/io/opentelemetry/instrumentation/lettuce/v5_1/LettuceTestUtil.groovy

```diff
@@ -16,12 +16,12 @@ final class LettuceTestUtil {
   static {
     def options = ClientOptions.builder()
     // Disable autoreconnect so we do not get stray traces popping up on server shutdown
-      .autoReconnect(false)
```

Comment:
I think you can delete this class as the java class you added is in the same package and has the same name